app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/dialogs/SendGeographicMessageDialog.java

```diff
@@ -125,8 +140,14 @@ protected boolean hasPositiveButton() {
     protected void onPositiveClick() {
```

Comment:
As a code reader, this method does a lot of different things that I need to understand in-order to see what the onPositiveClick method does. One way to resolve it, is to extract some of its "inner-workings" to different methods.
It doesn't matter that its just short validation and sending, its still one method that does handles both hjgh-level flow decisions and low-level details of what makes user-input invalid.
Its harder to read, its harder to maintain  and therefore should be split into methods.